bin/ember-template-lint.js

```diff
@@ -67,14 +67,18 @@ function getRelativeFilePaths() {
 
   var relativeFilePaths = fileArgs
     .reduce((filePaths, fileArg) => {
-      return filePaths.concat(
-        globby.sync(fileArg, {
+      var files;
+      if (fileArg === '-' || fileArg === '/dev/stdin') {
+        files = ['/dev/stdin'];
+      } else {
+        files = globby.sync(fileArg, {
           ignore: ['**/dist/**', '**/tmp/**', '**/node_modules/**'],
           gitignore: true,
-        })
-      );
+        });
+      }
+      return filePaths.concat(files);
     }, [])
-    .filter(filePath => filePath.slice(-4) === '.hbs');
+    .filter(filePath => filePath === '/dev/stdin' || filePath.slice(-4) === '.hbs');
```

Comment:
I think we can move this filter up into the `else` of the conditional just above, and then we won't need to have `filePath === '/dev/stdin' ||` (since the `else` branch is only ran when we **aren't** using `/dev/stdin` AFAICT).